project.clj

```diff
@@ -92,6 +104,7 @@
                          [puppetlabs/jdbc-util "1.0.0"]
                          [puppetlabs/typesafe-config "0.1.5"]
                          [puppetlabs/ssl-utils "0.8.3"]
+                         [puppetlabs/jruby-utils "0.8.0"]
```

Comment:
Is there some other way we can resolve this?  I think jruby-utils is only being used by puppetserver today so I think it would be nice to avoid including it in clj-parent for now if we can.  @aperiodic - didn't you work this one out already recently?